tests/users/test_validation_request_data.py

```diff
@@ -196,7 +196,7 @@ def test_user_registration_request_data_name_with_just_spaces_good_length(self):
     def test_user_password_change_request_password_inferior_to_limit(self):
         secure_random = SystemRandom()
         random_generated_password = "".join(secure_random.choice(ascii_lowercase) for x in range(PASSWORD_MIN_LENGTH-1))
-        data = dict(new_password=random_generated_password, current_password=random_generated_password)
```

Comment:
Can you please remove the change in these file? Or do you think its really needed? @IlyaMaier

Reply:
@isabelcosta I must change that. Otherwise, the test will fail as the new and the current passwords are the same. Here is the screenshot:

![Screenshot (126)](https://user-images.githubusercontent.com/34242059/73528635-8aafcb00-4415-11ea-8db8-4db8f1f0b551.png)

Reply:
got it! I took some time to understand what you meant, but now I do 🙈 thank you for enlighting me, your change makes sense!